src/SAWScript/Crucible/LLVM/X86.hs

```diff
@@ -277,7 +277,7 @@ crucible_llvm_verify_x86 bic opts (Some (llvmModule :: LLVMModule x)) path nm gl
         ]
 
       liftIO $ printOutLn opts Info "Examining specification to determine preconditions"
-      methodSpec <- buildMethodSpec bic opts llvmModule nm (show addr) setup
+      methodSpec <- buildMethodSpec bic opts {- TODO! checkSat -} llvmModule nm (show addr) setup
```

Comment:
`_checkSat` is in scope

Reply:
True, but the code otherwise ignores it, and the haddocks explicitly claim it isn't used.  I don't know offhand if there is a deep reason path satisfiability was never hooked up for the X86 verification methods, or if it is only that we never got to it.

Reply:
let's hook it up, there isn't a reason not to use it